### PR TITLE
Improves visual of insert link menu

### LINKED ISSUE #256: Insert Link Menu Improvement
### Please describe the problem (or idea)
There's a blank space, but the text area for the URL doesn't occupy all the width of the menu
capturar2
It would be better if the text area occupied the full width, like in the Attach File menu. It should also have the highlight and rounded edges.
capturar

### Please show us where to look

Just click on the Hyperlink and Attachment buttons in section 4.

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by IgorWilbert**
Fixes #256 

done2

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [X] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt jasmine`
* [X] code is in uniquely-named feature branch and has no merge conflicts
* [X] PR is descriptively titled
* [X] PR body includes `fixes #0000`-style reference to original issue #
* [X] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by rexagod** [comment]
Nice work, Igor! Just one little thing, can you make [these](https://user-images.githubusercontent.com/42987854/51743701-ab1bab80-2084-11e9-9680-549d59ed8313.JPG) "cancel" and "ok" buttons resemble the ones [here](https://user-images.githubusercontent.com/42987854/51698168-ae655780-1ff0-11e9-926a-765d86740410.JPG) (maybe lighten up the button borders)?

**Comment by IgorWilbert** [comment]
Sure @rexagod , is it good now? It looks good here.

**Comment by rexagod** [comment]
Hey can you share a screenshot? Just want to be sure!

**Comment by IgorWilbert** [comment]
@rexagod I already updated the PR's screenshot, I thought you would notice

**Comment by IgorWilbert** [comment]
I think this should solve the issue. @jywarren please review, thanks!

File: dist/PublicLab.Editor.css [hunk 1]
```diff
@@ -173,6 +173,43 @@ textarea.ple-textarea {
...
+  border: 1px solid #DDDD;
+  height: 30px;
+  text-indent: 15px;
+  box-shadow: 0 0 5px rgba(81, 203, 238, 1);
+}
+
+.wk-prompt-cancel {
+  border-radius: 5px;
+  border: 1px solid #A9A9A9;
+  height: 30px;
```
Comment (VladimirMikulic) [inline dist/PublicLab.Editor.css]: This line should be removed.

File: dist/PublicLab.Editor.css [hunk 2]
```diff
@@ -173,6 +173,43 @@ textarea.ple-textarea {
...
+}
+
+.wk-prompt-cancel:active{
+  box-shadow: 0 0 5px rgba(0, 0, 0, 0.5);
+}
+
+.wk-prompt-ok {
+  border-radius: 5px;
+  border: 1px solid #A9A9A9;
+  height: 30px;
```
Comment (VladimirMikulic) [inline dist/PublicLab.Editor.css]: This one as well.

**Comment by nicoleiocana** [comment]
This is incorrect. The classes are to be updated in the `dist/PublicLab.Editor.js` file. Specifically, on lines 19788 & 19789 respectively. The update that upholds the DRY principal is to update the code to this:

```
    cancel: e('button', 'wk-prompt-cancel btn btn-default', 'Cancel'),
    ok: e('button', 'wk-prompt-ok btn btn-default', 'Ok'),
```

The styling issues that @VladimirMikulic brought up are already handled by bootstrap 4, which the plots2 repo utilizes.

**Comment by VladimirMikulic** [comment]
@nicoleiocana your path is incorrect. We don't directly update the dist file :)

**Comment by nicoleiocana** [comment]
You are correct. It was bad for me to assume that the contributors understood where the files existed in the code. I admit my mistake.

We edit the file here:

`/PublicLab.Editor/node_modules/woofmark/src/prompts/render.js` lines 30 & 31.

**Comment by VladimirMikulic** [comment]
@nicoleiocana we never modify files in node_modules. This is highly discouraged and anti-pattern!

**Comment by jywarren** [comment]
Hi, all! I think i understand that Bootstrap 4 in `plots2` makes this unnecessary, but perhaps we should update this repo to Bootstrap 4 so that the ambiguity is gone? In the meantime, I'll close this up. Thank you all!!!

**Pull request closed without merging**